src/routes/accommodation.route.js

```diff
@@ -118,4 +118,69 @@ router.get('/:accomodationId', verifyToken.headerToken, verifyUser, Accommodatio
  *            type: string
  */
 router.post('/booking', Validate.bookingValidation(), isValid, verifyToken.headerToken, AccomodationMiddleware.validateDates, AccomodationMiddleware.checkBookingFacilitiesAvailability, Accommodation.bookAccommodation);
+router.post('/rate/:accomodationId', verifyToken.headerToken, verifyUser, AccomodationMiddleware.checkValidAccomodationRates, AccomodationMiddleware.checkUserAccomodationCorrespondance);
+/**
+ * @swagger
+ *
+ * /accommodations/rate/{accomodationId}:
+ *    post:
+ *      summary: user should be to rate an accommodation
+ *      tags: [Accommodations]
+ *      parameters:
+ *       - name: accomodationId
+ *         in: path
+ *         description: rate that specific accomodation
+ *         required: true
+ *         type: string
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/rate'
+ *      responses:
+ *        "200":
+ *           description: rate create successfully
+ *
+ * components:
+ *    schemas:
+ *      rate:
+ *        type: object
+ *        required:
+ *          - rate
+ *        properties:
+ *          rate:
+ *            type: integer
+ */
+
+router.post('/rate/:accomodationId', verifyToken.headerToken, verifyUser, AccomodationMiddleware.checkValidAccomodationRates, AccomodationMiddleware.checkUserAccomodationCorrespondance, Accommodation.rateAccomodation);
+/**
+ * @swagger
+ *
+ * /accommodations/average-rate/{accomodationId}:
```

Comment:
Always the collection is followed by the elementID. Therefore, this should be /accommodations/accommodationID/ratings